docs/running-tests/chrome-chromium-installation-detection.md

```diff
@@ -0,0 +1,100 @@
+# Installation and Detection Protocol of Browser Binaries and Webdrivers for Chrome and Chromium
+
+This is a detailed description of the process in which `wpt` detects
+and installs the browser components for Chrome and Chromium.
+This process can seem convoluted and difficult to understand at first glance,
+but the reason for this process is to best ensure these components are
+compatible with each other and are the intended items that the user
+is trying to test.
+## Chrome
+
+### Detection
+- **Browser Binary**
```

Comment:
Let's drop the bullet lists. The bolded text already makes the structure clear, and notes in the middle of the second bullet list don't work.

```suggestion
**Browser**
```